### PR TITLE
EOS-23637: Manifest integration

### PR DESCRIPTION
**Pull request by suryakumar1024**
# Problem Statement
- Adding logs for manifest component support bundle

# Design
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/316735509/Support+Bundle+Framework

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/support_framework/bundle_generate.py [hunk 1]
```diff
@@ -161,13 +161,22 @@ async def init(command: List):
             else:
                 components_list = list(command_files_info.keys())
                 components_list.remove(const.SOS_COMP)
-        Log.debug(
-            f"Generating for {const.SB_COMPONENTS} {' '.join(components_list)}")
+        Log.debug(f"Generating for {const.SB_COMPONENTS} " \
+            f"{' '.join(components_list)}")
```
Comment (sachinpunadikar) [inline py-utils/src/utils/support_framework/bundle_generate.py:165]: Shouldn't we add manifest here, something like below:
Log.debug(f"Generating for  manifest and {const.SB_COMPONENTS} " \

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
See if suggestion is meaningful

File: py-utils/src/utils/support_framework/bundle_generate.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/support_framework/bundle_generate.py:165]: Updated in recent commit.

**Review by ujjwalpl — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**